bundles/CoreBundle/src/Migrations/Version20240708083500.php

```diff
@@ -34,6 +34,15 @@ public function getDescription(): string
      */
     public function up(Schema $schema): void
     {
+        // Delete old Version Name
+        $this->addSql('DELETE FROM `migration_versions` WHERE `migration_versions`.`version` = \'Pimcore\\\\Bundle\\\\CoreBundle\\\\Migrations\\\\Version20210107103923\'');
```

Comment:
One potential problem I just encountered is, that the table name is actually configurable. So it could be that it is not `migration_versions` which leads to errors.
@blankse @brusch

Reply:
This is unfortunately not super simple to solve. 
You'd need access to `doctrine.migrations.storage.table_storage` service that holds the ´related config. 

This is the way it works: 
https://symfony.com/bundles/DoctrineMigrationsBundle/current/index.html#migration-dependencies

But it also says: 
> The interface Symfony\Component\DependencyInjection\ContainerAwareInterface has been deprecated in Symfony 6.4 and removed in 7.0. If you use this version or newer, there is currently no way to inject the service container into migrations.

So I'd wait until there's proper future proof solution